pkg/client/unversioned/clientcmd/api/latest/latest.go

```diff
@@ -40,12 +40,24 @@ const OldestVersion = "v1"
 // with a set of versions to choose.
 var Versions = []string{"v1"}
 
-var Codec runtime.Codec
+var (
+	Codec  runtime.Codec
+	Scheme *runtime.Scheme
+)
 
 func init() {
-	yamlSerializer := json.NewYAMLSerializer(json.DefaultMetaFactory, api.Scheme, api.Scheme)
+	Scheme = runtime.NewScheme()
+	if err := api.AddToScheme(Scheme); err != nil {
```

Comment:
This one is really weird and I'm not entirely sure what's going on. Don't want to make more drastic changes at the moment.